UPGRADE.md

```diff
@@ -17,6 +17,49 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-alpha4 to Unreleased]
 
+### [shopsys/framework]
+#### PostgreSQL upgrade:
+We decided to move onto a newer version of PostgreSQL.
+
+These steps are for migrating your data onto newer version of postgres and are inspired by [official documentation](https://www.postgresql.org/docs/10/static/upgrading.html):
+
+If you are running your project natively then just follow [official instructions](https://www.postgresql.org/docs/10/static/upgrading.html), 
+if you are using docker infrastructure you can follow steps written below.
+
+1. create a backup of your database by executing::
+
+    `docker exec -it shopsys-framework-postgres pg_dumpall > backupfile`
+
+1. copy new version of `docker-compose.`:
+
+    `cp docker/conf/docker-compose.yml.dist -f docker-compose.yml`
+
+    *Note: select correct `docker-compose` according to your operating system*
+
+1. update version of `database_server_version` from *9.5* to *10.4* in your `parameters.yml`
+
+1. stop containers and delete old data:
+
+    `docker-compose down`
+
+    `rm -rf <project-root-path>/var/postgres-data`

```

Comment:
Change this to `rm -rf project-base/var/postgres-data/*` so it works even on Mac with docker-sync

Reply:
Oh, @MattCzerner I am so sorry, I oversaw what I was writing. I wanted to change the end of the command and add `/*` not to change beginning...